### ISSUE TITLE
Cloud service

### ISSUE THREAD

**Issue by develar** [GitbookIO/nuts#29]
It will be cool to setup cloud service. So, any developer (github account) can use auto update server in seconds without any efforts.

It is safe since you should sign your app in any case (so, even if server will be compromised, users will not be affected because OS will just block unsigned/unidentified app).

Is there particular reason why do you stream artifact (and cache it) instead of just redirect to github?

I am going to implement redirection instead of streaming and launch such service.

**Comment by imlucas** [comment]
@develar I believe it's because redirects won't work for private repos.

**Comment by develar** [comment]
@imluca Yes, now there is a way to disable proxy — https://github.com/electron-userland/electron-builder/issues/486#issuecomment-225415972
